README.md

```diff
@@ -95,16 +95,32 @@ POST body (as JSON):
 
 ## Testing
 
-Folder test, contains a JSON data file and a `curl-format.txt`, usefull for testing. In the example bellow we used `service.sh develop` to run a local server on port `8000`.
+Folder test, contains a JSON data files, `curl-format.txt`, and `test_analysis_endpoint.py`, usefull for testing. In the example bellow we used `service.sh develop` to run a local server on port `5020`.
 
 ```bash 
 # At project root level
-./service.sh develop
+./analysis.sh develop
```

Comment:
this seems to have been correct the way it was. I can't seem to find a analysis.sh file in the project source code. can you clarify this change, please?